common/config/rush/common-versions.json

```diff
@@ -65,7 +65,7 @@
     // projects to choose a higher version if desired.
     "@azure/core-http": ["^1.0.0"],
     // @azure/container-registry is using the beta.1 version for CAE support
-    "@azure/core-rest-pipeline": ["1.1.0-beta.1"],
+    "@azure/core-rest-pipeline": ["^1.0.3", "1.1.0-beta.1"],
```

Comment:
What is the other version being used other than these two? Maybe that one should be listed here as an alternative as it feels that `^1.0.3` should be the common one while others are exceptions?

Reply:
right now core-rest-pipeline has 1.1.0-beta.2 in its package.json but it is unpublished so I can not use it as a dep.

Reply:
actually I misunderstood your question. There is no other versions used so I just removed the exception for 1.0.3.